persistence.go

```diff
@@ -0,0 +1,131 @@
+package presentswitch
+
+import (
+	"database/sql"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+// Represents the current state of a RoomInfo.
+type RoomInfo struct {
+	ID             int // room number
+	RoomName       string
+	CurrentTitle   string
+	CurrentSpeaker string
+	CurrentTime    string
+	NextTitle      string
+	NextSpeaker    string
+	NextTime       string
+}
+
+// InitDB creates a new DB object using filename as parameter
+func InitDB(filepath string) *sql.DB {
+	db, err := sql.Open("sqlite3", filepath)
+	if err != nil {
```

Comment:
We can shrink this `if` to something briefer and more readable:

```
if err != nil; db == nil {
...
}
```

Reply:
Sure. I'll do it

Reply:
I leave it like that because of the other lines:
```
        db, err := sql.Open("sqlite3", filepath)
	if err != nil {
		panic(err)
	}
	if db == nil {
		panic("db nil")
	}
```
I could shrink the first if, but not the second (or can I?)

Reply:
Ok, I understand your point. If you merge both ifs you will not be able to tell which error is which.